### ISSUE TITLE
Insulated Conduits not connecting to RFTools machines

### ISSUE THREAD

**Issue by andrejsavikin** [SleepyTrousers/EnderIO#3389]
#### Issue Description:
Insulated redstone conduit isn't connecting to some RFTools machines. Machines confirmed not working: Endergenic Generator, Pearl Injector. Machines confirmed working: Ender Monitor. All machines listed previously work fine when connected with vanilla redstone

#### What happens:
I was setting up my endergenic generator setup and EIO wires didn't connect to my RFTools machines.
(Edit: the setup in question is http://s.savikin.me/scr/javaw_2016-06-08_12-16-59.jpg )

#### What you expected to happen:
Conduits connecting to the machines and passing redstone signals as needed.

#### Steps to reproduce:

1. Install EIO
2. Install RFTools
3. Place uninsulated conduit next to endergenic generator
...

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO: 1.9.4-0.4.0.23-alpha
- EnderCore: 1.9.4-0.4.0.23-alpha
- Minecraft: 1.9.4
- Forge: 12.14.0.1951

#### Your most recent log file where the issue was present: 
This happened on a server where I don't have access to logs, so it's impossible for me to provide logs

**Comment by HenryLoenwind** [comment]
Are you connecting the conduits or are you just placing them? Insulated Redstone conduits need to be connected, that's what makes them insulated...

**Comment by andrejsavikin** [comment]
I am not, since I am not able to connect them to machines with rclick with yeta wrench. I thought this happens automatically

**Comment by HenryLoenwind** [comment]
Some vanilla blocks are whitelisted fro auto-connect.

So this is odd, you should be able to connect a redstone conduit to everything, even air. Are they in a facade or not? With facades on servers there is a known issue, without it would be something new.

**Comment by andrejsavikin** [comment]
No facades, I wasn't even able to connect it to a redstone lever. Will try connecting to other things soon

**Comment by andrejsavikin** [comment]
Also, looking at my mod list I'd say the version indicator ingame is broken too because it says I have 0.23_alpha, while I have 0.36_alpha. Sorry about that

**Comment by HenryLoenwind** [comment]
Maybe you looked at the version of endercore by mistake? Otherwise I'd like a screenshot, maybe there's something wrond on our end.

@CrazyPants ping, as you know those conduits best.

**Comment by andrejsavikin** [comment]
I think I see why the conduits aren't working as intended. Will elaborate tonight when I come back home

**Comment by andrejsavikin** [comment]
For of all, I'll say that this is most likely a user(my) error or unintended feature. 
I had my yeta wrench in redstone conduit mode. I was looking at a conduit at this angle http://s.savikin.me/scr/javaw_2016-06-08_21-03-45.png thinking it would expand if I right clicked it. Turns out you need to hit both conduit and whole block border from the same side, like so http://s.savikin.me/scr/javaw_2016-06-08_21-04-49.png . 
In conclusion, I think it would be easier to use if you only had to rclick conduit bounding box, instead whole block bounding box

**Comment by RealGrep** [comment]
Agreed, it's difficult to click the block's bounding box on the right side when there's a full block in the way you're trying to connect to.

**Comment by JCMais** [comment]
Something like the neotech connection gui would be wonderful. I don't have a screenshot right now, but you basically opens a gui which you can rotate the block and pick what side you want to change to what.

**Comment by tterrag1098** [comment]
Because someone broke the selection boxes. Conduits should not have full-size selection boxes.

**Comment by HenryLoenwind** [comment]
yes, they are broken. Crazy's not been available to fix them lately, and I am a bit afraid of that code, tbh.

**Comment by VT-14** [comment]
Isn't there a keybind combination to click on any part of a conduit to open a GUI that lets you pick any side of the conduit and open that side's GUI?  I thought it was Shift + Left Click (while holding a Yeta Wrench), but I had issues with it in 1.7 and haven't tried in 1.8+.

**Comment by tterrag1098** [comment]
Yes, that's it
